changelogs/fragments/595-aws_s3-add-latest-choice-on-overwrite-parameter.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- aws_s3 - add latest choice on overwrite parameter to get latest object on S3
```

Comment:
```suggestion
- aws_s3 - add latest choice on overwrite parameter to get latest object on S3 (https://github.com/ansible-collections/amazon.aws/pull/595)
```

Reply:
@alinabuzachis I fixed by 6bd6bee. Thanks for your suggestion.